### ISSUE TITLE
Sync not triggered for database added during setup

### ISSUE THREAD

**Issue by robdaemon** [metabase/metabase#12826]
**Describe the bug**
During the initial setup, you can add a database, but it isn't available until you force a sync in the Admin panel.

**To Reproduce**
1. During setup, add a new database.
2. Once setup is complete, attempt to browse that database via "Browse Data" or access the data model in the Admin panel - no tables will appear.
3. Go to the Databases section in Admin, select the newly added database, and click "Sync database schema now"
4. Wait until sync is complete
5. Refresh your browser to see the datamodel appear

**Expected behavior**
A user should be notified that the database is "empty" pending a sync when they attempt to view the datamodel or browse data (or some other mechanism for making it obvious that it's just not ready yet)

**Screenshots**

![Screenshot_20200701_162218](https://user-images.githubusercontent.com/287213/86301562-08415800-bbbb-11ea-8909-bbb9d72fb52d.png)
![Screenshot_20200701_162245](https://user-images.githubusercontent.com/287213/86301564-08d9ee80-bbbb-11ea-9406-a21dce82010c.png)
![Screenshot_20200701_162314](https://user-images.githubusercontent.com/287213/86301566-08d9ee80-bbbb-11ea-918d-d1ec960ef244.png)
![Screenshot_20200701_162412](https://user-images.githubusercontent.com/287213/86301569-09728500-bbbb-11ea-895d-fc6da45660be.png)

**Information about your Metabase Installation:**

```
{
  "browser-info": {
    "language": "en-US",
    "platform": "Linux x86_64",
    "userAgent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36",
    "vendor": "Google Inc."
  },
  "system-info": {
    "file.encoding": "UTF-8",
    "java.runtime.name": "OpenJDK Runtime Environment",
    "java.runtime.version": "11.0.7+10-post-Ubuntu-3ubuntu1",
    "java.vendor": "Ubuntu",
    "java.vendor.url": "https://ubuntu.com/",
    "java.version": "11.0.7",
    "java.vm.name": "OpenJDK 64-Bit Server VM",
    "java.vm.version": "11.0.7+10-post-Ubuntu-3ubuntu1",
    "os.name": "Linux",
    "os.version": "5.4.0-39-generic",
    "user.language": "en",
    "user.timezone": "UTC"
  },
  "metabase-info": {
    "databases": [
      "h2",
      "oracle"
    ],
    "hosting-env": "unknown",
    "application-database": "h2",
    "application-database-details": {
      "database": {
        "name": "H2",
        "version": "1.4.197 (2018-03-18)"
      },
      "jdbc-driver": {
        "name": "H2 JDBC Driver",
        "version": "1.4.197 (2018-03-18)"
      }
    },
    "run-mode": "dev",
    "version": {
      "tag": "v0.36.0-snapshot",
      "hash": "8c6d854",
      "branch": "oracle-enums",
      "date": "2020-06-30"
    },
    "settings": {
      "report-timezone": null
    }
  }
}
```

**Severity**
No data is lost, but it isn't a great user experience on first install.

**Comment by robdaemon** [comment]
This is related to #12825 (but not a duplicate of it)

**Comment by flamber** [comment]
But is it not running the sync, when the database is added during the setup?
If it is running the sync, but just haven't completed yet, then it's #11354.
Related to #10927.

**Comment by robdaemon** [comment]
It's not running the sync during setup. I had to go force it.
